src/main/resources/docker-compose/postgres/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '2.2'
+
```

Comment:
I make sense to add script that we apply some schema on startup.

Reply:
Added hello_world_schema.sql file
Added script for creating a schema to each Postgres instance